draft-bradleylundberg-cfrg-arkg.md

```diff
@@ -376,8 +376,8 @@ ARKG-Generate-Seed() -> (pk, sk)
     Inputs: None
 
     Output:
-        (pk, sk)  An ARKG seed key pair with public key pk
-                    and private key sk.
+        (pk, sk)  An ARKG seed pair with public seed pk
```

Comment:
As I noted in https://github.com/Yubico/arkg-rfc/pull/15, the seed pair is not a private, public pair but a private, shared secret pair. The shared secret seed must be kept secret between entities who are involved in the public key generation.

Reply:
Ah yes, that's a better term!

The sensitivity of the shared key depends on the application: knowing the shared key does not give an attacker any advantage against SK-security or PK-unlinkability as defined in [[Frymann2020](https://eprint.iacr.org/2020/1004)], but it can indeed give advantage in other kinds of attacks.

Reply:
Changing the term throughout the document will probably cause conflicts with a bunch of other PRs, so let's handle that in a separate PR.